core/src/test/scala/org/elasticmq/actor/QueueActorMsgOpsTest.scala

```diff
@@ -130,15 +130,15 @@ class QueueActorMsgOpsTest extends ActorTest with QueueManagerForEachTest with D
       lookupAfterReceiving <- queueActor ? LookupMessage(MessageId("xyz"))
     } yield {
       // Then
-      lookupBeforeReceiving.flatMap(_.deliveryReceipt) should be (None)
+      lookupBeforeReceiving.flatMap(_.deliveryReceipt) should be(None)
 
       val receivedReceipt = received.flatMap(_.deliveryReceipt)
       val lookedUpReceipt = lookupAfterReceiving.flatMap(_.deliveryReceipt)
 
-      receivedReceipt.size  should be > (0)
-      lookedUpReceipt should be ('defined)
+      receivedReceipt.size should be > (0)
```

Comment:
[minor] there's quite a lot of changes from reformatting which make it harder to spot the "real" changes